lms/views/api/sync.py

```diff
@@ -11,6 +11,7 @@ class LMS(Schema):
         product = fields.Str(required=True)
 
     lms = fields.Nested(LMS, required=True)
+    assignment_id = fields.Int(required=True)
```

Comment:
This being required means all in-flight request will fail. It's probably not a lot but we'll have to schedule the deploy early morning.

Alternately we could make this optional and have a conditional further down to do the data collection based on that.

Reply:
Yep. The commits are made so we can split them up if required and we will start sending it first. Although I'm wondering how how we handle present but not expected values. So more work might need to be done.

I wanted to have a chat with you about how much this is worth working around vs. a yolo approach.

Reply:
We've had a chat and decided that today is a good day to deploy this, as most users are on holiday.